eng/native/configurepaths.cmake

```diff
@@ -2,3 +2,5 @@ get_filename_component(CLR_REPO_ROOT_DIR ${CMAKE_CURRENT_LIST_DIR}/../.. ABSOLUT
 set(CLR_ENG_NATIVE_DIR ${CMAKE_CURRENT_LIST_DIR})
 get_filename_component(CLR_SRC_NATIVE_DIR ${CMAKE_CURRENT_LIST_DIR}/../../src/native ABSOLUTE)
 get_filename_component(CLR_SRC_LIBS_NATIVE_DIR ${CMAKE_CURRENT_LIST_DIR}/../../src/libraries/Native ABSOLUTE)
+set (CLR_ARTIFACTS_OBJ_DIR "${CLR_REPO_ROOT_DIR}/artifacts/obj")
```

Comment:
👍 we can also use it here:
https://github.com/dotnet/runtime/blob/1cd5033bb37d34aecacabb7a4a1543774961a74e/src/libraries/Native/Windows/System.IO.Compression.Native/CMakeLists.txt#L87-L88
(although that .rc could also be defined in one place as `VERSION_RC_FILE_PATH`)